CalibMuon/DTCalibration/plugins/DTTTrigCorrection.cc

```diff
@@ -26,7 +26,7 @@
 #include "CalibMuon/DTCalibration/interface/DTTTrigCorrectionFactory.h"
 #include "CalibMuon/DTCalibration/interface/DTTTrigBaseCorrection.h"
 #include "FWCore/Framework/interface/ConsumesCollector.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
```

Comment:
@yuanchao You don't need this one, since it is already in the header file (DTTTrigCorrection.h)

Reply:
@yuanchao actually, if you remove it, that would be also remove the conflict with my PR :-)

Reply:
@yuanchao was this remove after all? Or did @ggovi remove it from his code?